src/handlers/imgix.ts

```diff
@@ -48,28 +52,40 @@ interface ImgOptions {
   fm?: string;
 }
 
+// We use this to reduce our quota for Cloudinary and Imgix
+const roundUpTo50 = (value: number) => Math.ceil(value / 50) * 50;
+
 const shouldSignUri = (
   externalImageUri: string,
   options?: ImgOptions
 ): string | undefined => {
   try {
+    const updatedOptions: ImgOptions = {
+      ...(options?.w && {
+        w: roundUpTo50(PixelRatio.getPixelSizeForLayoutSize(options.w)),
+      }),
+      ...(options?.h && {
+        h: roundUpTo50(PixelRatio.getPixelSizeForLayoutSize(options.h)),
+      }),
+      ...(options?.fm && {
+        fm: options.fm,
+      }),
```

Comment:
since we run this code pretty much fun rneqxh image can we refactor it to be just a bunch of "if" and and options.key assignment since it's gonna be faster.